lib/cartopy/img_transform.py

```diff
@@ -13,11 +13,11 @@
 
 
 try:
-    import pykdtree.kdtree
+    from pykdtree.kdtree import KDTree as _kdtree_handle
```

Comment:
```suggestion
    from pykdtree.kdtree import KDTree as _kdtreeClass
```
Thoughts on making this explicit that it is a Class you're using below? I wasn't entirely clear what handle meant at first glance.

Reply:
also done!